packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/default.yml

```
@@ -17,6 +14,15 @@ processors:
       field: observer.type
       value: firewall
 
+# Set timezone, default to local.
+  - set:
+      field: event.timezone
+      value: "+0000"
+  - set:
+      field: event.timezone
+      value: "{{_conf.tz_offset}}"
```

```suggestion
      value: "{{{_conf.tz_offset}}}"
```

to prevent any HTML escaping being added by the [mustache](https://mustache.github.io/mustache.5.html) template engine in ES.

---

Does this also need to happen elsewhere with `event.timezone` (below).

---

I use the triple-brace version everywhere. I can't think of a use-case where I would want HTML escaping added.

And I try to avoid templates where possible. In the case of `set` it's usually better to use `copy_from`.

---

The specific place I'm thinking of is the date processor
```
  - date:
      field: "_temp_.generated_time"
      formats:
        - "yyyy/MM/dd HH:mm:ss"
        - "strict_date_optional_time_nanos"
      timezone: "{{ event.timezone }}" # HERE
```